File: Project1/C++/src/ListNode.cpp
#include <string>
#include <iostream>
using namespace std;

struct ListNode
{
    string data;
    ListNode* next;

    ListNode(string s)
    {
        data = s;
        this->next = NULL;
    }

    ListNode(string s, ListNode* n)
    {
        data = s;
        this->next = n;
    }
};

File: Project1/Java/src/ListNode.java

public class ListNode
{
    public String data;
    public ListNode next;

    public ListNode()
    {
        this(null);
    }

    public ListNode(String e)
    {
        data = e;
        next = null;
    }

    @Override
    public String toString() { return data; }
}